src/trackers/types.ts

```diff
@@ -45,8 +44,8 @@ export interface ITelemetryTracker {
 }
 
 export interface IFilterAdapter {
-  add(featureName: string, key: string): boolean;
-  contains(featureName: string, key: string): boolean;
+  add(key: string, featureName: string): boolean;
```

Comment:
can we keep the signature of the track method as the original? track(featureName: string, key: string),